es-vue-base/src/lib-components/EsNavBarLink.vue

```diff
@@ -0,0 +1,31 @@
+<template>
```

Comment:
Is there anything unique to the nav-bar? I guess I wonder if relative links defaulting to `:to` vs. absolute defaulting to `:href` could be used beyond the nav i.e. could this be an `EsLink` component?

Reply:
I could see an `EsLink` component being used internally to ESDS in cases where an exported component contains links, the behavior of which, downstream applications may want to modify. There probably aren't too many cases of this in ESDS. Nav bar and footer are a bit of a special case that won't live within ESDS for too much longer anyway, but definitely something to keep in mind for the future if we ever find another case within ESDS.

It's also a very opinionated behavior that may not work in all cases - there are probably cases where you'd want to allow root-relative URLs to be normal `href` attributes, if an application was only on one domain, wasn't an SPA, and that's how it wanted to specify its URLs. But that does seem unlikely in our new tech stack, at least for the moment.